package.json

```diff
@@ -6,7 +6,7 @@
     "Alex Ehrnschwender <alexanderscott46@gmail.com>",
     "Matthew Scragg <scragg@gmail.com>"
   ],
-  "name": "redis-commander",
+  "name": "@awearsolutions/redis-commander",
```

Comment:
@ozomer is this correct package name? I'm asking because it was merged to `master`.

Reply:
Yes, I have cloned the project to [here](https://bitbucket.org/awearsolutionsltd/redis-commander/overview) because I needed some fixes for some internal project.
Feel free to merge the code (I don't know about the required git commands, but I guess copy-pasting the code will do the trick).